src/apb/engine.py

```diff
@@ -433,8 +433,8 @@ def broker_resource_url(host, broker_name):
 def relist_service_broker(kwargs):
     try:
         openshift_config.load_kube_config()
-        token = openshift_client.configuration.api_key['authorization']
-        cluster_host = openshift_client.configuration.host
+        token = openshift_client.Configuration().get_api_key_with_prefix('authorization')
```

Comment:
This also exists on line 1092.

Reply:
Thanks. Updated the PR message and fixed 1092.